examples/csi_driver_config_vars.yaml

```diff
@@ -0,0 +1,12 @@
+---
+service_instance_id: "" #IBM Cloud PowerVS service instance id
+region: ""              #IBM Cloud PowerVS service instance region
+zone: ""                #IBM Cloud PowerVS service instance zone
+
+masters:
+  - name: ""            #OCP Nodes Master name
+    id: ""              #OCP Nodes Master instance id
+
+workers:
+  - name: ""            #OCP Nodes Worker name
+    id: ""              #OCP Nodes Worker instance id
```

Comment:
Can we have some example values in it? May be with multiple master nodes as well.

Reply:
Done.